scipy/stats/stats.py

```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from
            normality", Biometrika, 60, 613-622
 
+    Examples
+    --------
+    >>> import scipy.stats as stats
+    >>> pts = 1000
+    >>> a = np.random.multivariate_normal([0], [[1]], size=pts)
+    >>> b = np.random.multivariate_normal([2], [[1]], size=pts)
+    >>> x = np.concatenate((a, b))
+    >>> k2, p = stats.normaltest(x)
```

Comment:
I see that you added a call to set the seed.  Were you also going to add `print(p)`?